.github/workflows/format.yml

```
@@ -15,7 +15,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v2
         with:
-          python-version: 3.9
+          python-version: 3.12

```

```suggestion
          python-version: "3.10"
```

I think you want to use 3.10 instead.